src/application/components/Explorer/GraphRefModal.tsx

```diff
@@ -29,207 +33,88 @@ export const GraphRefModal = ({
   const [graphVariant, setGraphVariant] = useState<string>();
 
   return (
-    <>
-      <div
-        css={{
-          width: "100vw",
-          height: "100vh",
-          position: "absolute",
-          backgroundColor: "rgb(25, 28, 35)",
-          opacity: "0.55",
-        }}
-      ></div>
-      <div
-        css={{
-          width: 470,
-          height: 400,
-          position: "absolute",
-          margin: "auto",
-          top: "20%",
-          left: "calc(50% - 250px)",
-          display: "flex",
-          backgroundColor: "white",
-          borderRadius: 12,
-          boxShadow:
-            "rgb(0 0 0 / 12%) 0px 16px 32px 0px, rgb(18 21 26 / 4%) 0px 0px 0px 1px",
-          fontFamily: '"Source Sans Pro", sans-serif',
-          fontSize: 15,
-          textAlign: "center",
-        }}
+    <Modal open onClose={onClose} size="lg" className="h-[422px]">
```

Comment:
To be honest, I'm not entirely sure. I'm not in love with the existing functionality and resorted to forcing it to open in my work so that I could update it.

I would love to revisit this in a future PR, but updating what was already there felt a bit out of scope for this PR as it has already grown pretty large.